doc/classes/TileSet.xml

```diff
@@ -113,7 +113,7 @@
 			<argument index="0" name="source_from" type="int" />
 			<argument index="1" name="coords_from" type="Vector2i" />
 			<description>
-				Returns the coodinate-level proxy for the given identifiers. The returned array contains the two proxie's target identifiers (source ID and atlas coords ID).
+				Returns the coordinate-level proxy for the given identifiers. The returned array contains the two proxie's target identifiers (source ID and atlas coords ID).
```

Comment:
We can fix the other typo on the same line while we're at it :slightly_smiling_face: 

```suggestion
				Returns the coordinate-level proxy for the given identifiers. The returned array contains the two proxies' target identifiers (source ID and atlas coordinates ID).
```

Reply:
Given the sentence, wouldn't this be "the two proxy target identifiers"?

Reply:
Yes, I think you're correct @octetdev2. Another option is "the two target identifiers of the proxy".

Please remember to squash the commits so it's ready to merge.